Guidance for kid header (#77)
decentralized-identity/jwt-vc-presentation-profile: spec/spec.md

[[ref: VC Data Model v1.1]] specifies that "issuanceDate" property MUST be represented as an `nbf` JWT claim, and not `iat` JWT claim. This might sound couterintuitive, but the implementers of this profile MUST follow this guidance.

#### `kid` JWT header

::: note
  The requirement to use absolute or relative URIs within the `kid` header is currently under debate.
  VCs issued with this profile currently use an absolute URI.

  Future versions of the specifications or profile may update this guidance and requirements for the `kid` header.
  Due to this debate (and the potential for transition), implementors are advised to consider accepting both absolute and relative URIs.

  In this case of a relative `kid` URI, the `kid` is treated as being relative to the issuer URI contained within `iss`.
:::

> In the case of an absolute `kid` URI, the `kid` MUST be a URI that is prefixed by the issuer URI contained within `iss`.

Reviewer: ```suggestion
With an absolute `kid` URI, the `kid` MUST be a URI that is prefixed by the issuer URI contained within `iss`.
```

> 
> ::: note
>   The requirement to use absolute or relative URIs within the `kid` header is currently under debate.
>   VCs issued with this profile currently use an absolute URI.
> 
>   Future versions of the specifications or profile may update this guidance and requirements for the `kid` header.
>   Due to this debate (and the potential for transition), implementors are advised to consider accepting both absolute and relative URIs.
> 
>   In this case of a relative `kid` URI, the `kid` is treated as being relative to the issuer URI contained within `iss`.
> :::
> 
> In the case of an absolute `kid` URI, the `kid` MUST be a URI that is prefixed by the issuer URI contained within `iss`.
> 

Reviewer 2: ```suggestion

::: note
  The requirement to use absolute or relative URIs within the `kid` header is currently under debate.
  VCs issued with this profile currently use an absolute URI.

  Future versions of the specifications or profile may update this guidance and requirements for the `kid` header.
  Due to this debate (and the potential for transition), implementors are advised to consider accepting both absolute and relative URIs.

  In this case of a relative `kid` URI, the `kid` is treated as being relative to the issuer URI contained within `iss`.
:::

In the case of an absolute `kid` URI, the `kid` MUST be a URI that is prefixed by the issuer URI contained within `iss`.

  In the case when `kid` is a relative DID URL, the `kid` is a relative DID URL that does not start with the issuer URI contained within `iss` Claim.
```

Reviewer: This sounds great.  @troyronda - if you're good with it, we commit/ merge!
